lib/mongo/collection/view/builder/aggregation.rb

```diff
@@ -93,7 +93,10 @@ def write?
           def aggregation_command
             command = BSON::Document.new(:aggregate => collection.name, :pipeline => pipeline)
             command[:cursor] = cursor if cursor
-            command[:readConcern] = collection.read_concern if collection.read_concern
+            if collection.read_concern
+              command[:readConcern] = Options::Mapper.transform_values_to_strings(
```

Comment:
All of these read concern changes don't seem related to the ticket at all; are they required for correctness, or is it possible to just make the write concern changes and make these read concern changes in a separate commit?

Reply:
I removed read concern changes from this PR.